state/syncer/userAccountsSyncer.go

```diff
@@ -129,7 +128,7 @@ func (u *userAccountsSyncer) SyncAccounts(rootHash []byte) error {
 
 	log.Debug("main trie synced, starting to sync data tries", "num data tries", len(u.dataTries))
 
-	err = u.syncAccountDataTries(mainTrie, tss, ctx)
+	err = u.syncAccountDataTries(mainTrie, ctx)
```

Comment:
after we finished the trie sync, on L137, for example, we need to somehow call a reset method on the syncStatisticsHandler in order to not broadcast the last value from the last trie sync.